gui/default/syncthing/core/syncthingController.js

```diff
@@ -629,6 +631,14 @@ angular.module('syncthing.core')
             }).error($scope.emitHTTPError);
         }, 2500);
 
+        var refreshLastLocalChanges = debounce(function () {
+            $http.get(urlbase + "/events/disk?limit=20").success(function (data) {
+                $scope.globalChangeEvents = data;
```

Comment:
I think this is a personal preference, I like it the way it is.  Half will want it one way half the other.